_posts/2015-06-09-Fostering-Creativity-with-Semantically-Rich-APIs.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Semantically Rich APIs
+excerpt: Fostering Creativity with Semantically Rich APIs
+layout: post
+date: 2015-06-09 00:00:00 +0200
+author: stephen
+published: true
+---
+
+In our recent blog post, Kyle Fuller [wrote about](http://blog.apiary.io/2015/06/04/Hyperdrive/) the benefits of decoupling semantics from implementation details for building better API clients. I'd like to explore how this decoupled approach also benefits the API design process—it's quite nice to decouple your mind from implementation details as well.
+
+## The Difficulty with Common Approaches
+
+Many modern approaches in designing and building APIs couple not only clients and servers to implementation details, but also couple thought, creativity, and shared understanding to that implementation. Coupling servers and clients is a problem that is easily seen, and in theory no one wants a coupled approach. Kyle Fuller wrote in his post:
```

Comment:
Going with Kyle. I think the formal way is to use last names, but that feels too informal for a blog post.